src/routes/group.js

```diff
@@ -1,11 +1,45 @@
+/**
+ * @swagger
+ * tags:
+ *    name: Group
+ *    description: API to manage groups.
+ */
+
 import express from 'express';
 import Group from '../controllers/group';
 
 const groups = express.Router();
 
-// TODO: Add Swagger Docs
-
-// add a new group
+/**
+ * @swagger
+ * path:
+ * /groups/:
+ *    post:
+ *      description: Creates a new group
+ *      summary: Creates a group
+ *      tags:
+ *        - Group
+ *
+ *      requestBody:
+ *        description:
+ *        required: true
+ *
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
```

Comment:
Properties are missing; group creation requires a `creator` field, which represents the ObjectId of the user that owns the group.

Reply:
Yes, that's in TODO. To references user, we need to create a user schema.

Reply:
I'll resolve the conversation, since it's in TODO